README.md

```diff
@@ -13,6 +13,39 @@ clusters on Kubernetes (K8s) powered by [Patroni](https://github.com/zalando/spi
 It is configured only through manifests to ease integration into automated CI/CD
 pipelines with no access to Kubernetes directly.
 
+### Operator features
+
+* Rolling updates on cluster changes
+* Volume resize without Pod restarts
+* Cloning Postgres clusters
+* Logical Backups to S3 Bucket
+* Standby cluster from S3 WAL archive
+* Configurable for non-cloud environments
+* UI to create and edit cluster manifests
+
+### PostgreSQL features
+
+* Supports PostgreSQL 9.3 to 12
```

Comment:
Spilo does not contain lower than 9.6 I think.

We should also not advertise, we can make it 9.6+ or 10+ to not having to deal with older versions.